
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../../course_schedule/">
      
      
        <link rel="next" href="../chapter_02/">
      
      
        
      
      
      <link rel="icon" href="../../assets/images/elixir_favicon.ico">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.1">
    
    
      
        <title>1. Introducing this course - Assess protein dynamics & post-translational modifications</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.484c7ddc.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Lato";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../stylesheets/extra.css">
    
      <link rel="stylesheet" href="../../stylesheets/neoteroi-mkdocs.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#we-have-the-structures-now-what-next" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
        <aside class="md-banner">
          <div class="md-banner__inner md-grid md-typeset">
            
            
This training material is under active development and considered pre-alpha.

          </div>
          
        </aside>
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Assess protein dynamics &amp; post-translational modifications" class="md-header__button md-logo" aria-label="Assess protein dynamics & post-translational modifications" data-md-component="logo">
      
  <img src="../../assets/images/white-orange-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Assess protein dynamics & post-translational modifications
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              1. Introducing this course
            
          </span>
        </div>
      </div>
    </div>
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/Bio2Byte/training-protein-dynamics-post-translational-modifications" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    Bio2Byte/training-protein-dynamics-post-translational-modifications
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



  

<nav class="md-nav md-nav--primary md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Assess protein dynamics &amp; post-translational modifications" class="md-nav__button md-logo" aria-label="Assess protein dynamics & post-translational modifications" data-md-component="logo">
      
  <img src="../../assets/images/white-orange-logo.png" alt="logo">

    </a>
    Assess protein dynamics & post-translational modifications
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Bio2Byte/training-protein-dynamics-post-translational-modifications" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    Bio2Byte/training-protein-dynamics-post-translational-modifications
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    About
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../course_schedule/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Course schedule
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Course Content
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Course Content
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    
  
    1. Introducing this course
  

    
  </span>
  
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    
  
    1. Introducing this course
  

    
  </span>
  
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#we-have-the-structures-now-what-next" class="md-nav__link">
    <span class="md-ellipsis">
      
        We have the structures now — what Next?
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="We have the structures now — what Next?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#where-should-we-focus-to-understand-protein-behavior" class="md-nav__link">
    <span class="md-ellipsis">
      
        Where should we focus to understand protein behavior?
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#11-introducing-the-training-methodology" class="md-nav__link">
    <span class="md-ellipsis">
      
        1.1 Introducing the training methodology
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#12-elixir-belgium-node-services-used-in-this-training" class="md-nav__link">
    <span class="md-ellipsis">
      
        1.2 ELIXIR Belgium node services used in this training
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="1.2 ELIXIR Belgium node services used in this training">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#121-introduction-to-bio2byte-tools" class="md-nav__link">
    <span class="md-ellipsis">
      
        1.2.1 Introduction to Bio2Byte Tools
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#122-introduction-to-scop3p-and-scop3ptm" class="md-nav__link">
    <span class="md-ellipsis">
      
        1.2.2 Introduction to Scop3P (and Scop3PTM)
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#13-rethinking-the-protein-structure-and-function" class="md-nav__link">
    <span class="md-ellipsis">
      
        1.3 Rethinking the protein structure and function
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="1.3 Rethinking the protein structure and function">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#131-proteins-are-dynamic-systems-not-static-objects" class="md-nav__link">
    <span class="md-ellipsis">
      
        1.3.1 Proteins are dynamic systems, not static objects
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#132-limitations-of-deep-learningbased-structure-prediction" class="md-nav__link">
    <span class="md-ellipsis">
      
        1.3.2 Limitations of deep learning–based structure prediction
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#133-memorization-and-dominant-conformations" class="md-nav__link">
    <span class="md-ellipsis">
      
        1.3.3 Memorization and dominant conformations
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_02/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    2. Introducing the biological context
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_03/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    3. Building biological profiles
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_04/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    4. Working with post-translational modifications
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_05/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    5. Combining PTM annotations to biophysical properties
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_06/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    6. Closing
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../follow_up_training/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Follow up training
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../references/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    References
  

    
  </span>
  
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
                  


  
  


  <h1>1. Introducing this course</h1>

<div><p>Welcome to this VIB training. The authors of this course designed a full teaching day that combines theory with hands-on activities, allowing participants to apply the concepts in practice.</p>
<div class="admonition question">
<p class="admonition-title">Section questions</p>
<ol>
<li>Why do we need this training ?</li>
</ol>
</div>
<hr>
<h2 id="we-have-the-structures-now-what-next">We have the structures now — what Next?</h2>
<h3 id="where-should-we-focus-to-understand-protein-behavior">Where should we focus to understand protein behavior?</h3>
<p>Proteins are often presented as static 3D structures, but in reality they are <strong>dynamic</strong>, context-dependent systems. Function does not emerge from a single snapshot—it arises from <strong>conformational landscapes</strong>, <strong>biophysical properties</strong>, and <strong>regulatory modifications</strong> that change across conditions, cell types, and time.</p>
<p>Post-translational modifications, such as phosphorylation, can reweight conformational states, alter interaction interfaces, and rewire signaling pathways without changing the underlying sequence. Likewise, mutations—whether disease-causing or benign—do not act uniformly: their impact depends on structural context, local dynamics, intrinsic disorder, and network connectivity within the protein.</p>
<p>This training bridges the gap between <strong>sequence, structure, and function</strong> by moving beyond static models to:</p>
<ul>
<li>interpret backbone dynamics, intrinsic disorder, and folding propensities</li>
<li>place PTMs and mutations in their <strong>structural and biophysical context</strong></li>
<li>understand how proteins sample <strong>multiple functional states</strong>, rather than a single “native” structure</li>
</ul>
<p>By integrating structural models, biophysical annotations, and experimental proteomics evidence, we gain a mechanistic view of protein function, one that is essential for interpreting signaling, disease mutations, and regulatory complexity in real biological systems.</p>
<h2 id="11-introducing-the-training-methodology">1.1 Introducing the training methodology</h2>
<p>The protein of interest for this training is the human <strong>proto-oncogene tyrosine-protein kinase receptor RET</strong>, encoded by the <strong>RET</strong> gene. 
This protein is a receptor tyrosine-protein kinase involved in numerous cellular mechanisms including cell proliferation, neuronal navigation, cell migration, and cell differentiation. It exits is multiple conformational states (phospho and non-phospho). You will learn more about it’s biological context in Chapter 2.</p>
<ol>
<li>We will predict biophysical features for this protein using the Bio2Byte online platform, specifically the <strong>B2BTools</strong> suite. To enable a deeper analysis of these biophysical properties, we will generate a multiple sequence alignment (MSA) of sequences sharing at least 90% identity. This alignment will be used to identify conserved and variable patterns across homologous sequences.</li>
<li>The MSA will be created using <strong>Clustal Omega</strong>, and the aligned kinase domain will be extracted using a <strong>Google Colab</strong> notebook.</li>
<li>Post-translational modifications (PTMs) for the protein of interest will be explored using the <strong>Scop3P</strong> online platform, which is directly linked from the B2BTools prediction results. After analyzing the available information on modifications, structures, and experimental evidence, we will follow a more detailed protocol to link biophysical patterns with PTMs.</li>
<li>Finally, the course will address the impact of mutations. We will show how to modify the wild-type sequence and assess the effect of single amino acid substitutions on biophysical profiles and predicted protein structures using <strong>AlphaFold v3</strong>.</li>
</ol>
<div class="admonition success">
<p class="admonition-title">Training material: Jupyter notebooks</p>
<p>Use the following link to launch the JupyterLab environment for this training:</p>
<p><a href="https://binder.compomics.com/v2/gh/Bio2Byte/Scop3P-notebooks/HEAD"><img alt="Launch Binder" src="https://mybinder.org/badge_logo.svg"></a></p>
<p>The environment may take a few minutes to build and start. Please open the link now so that it is ready when needed later in the training.</p>
</div>
<details class="notes">
<summary>User-interface example</summary>
<p>This is how the interface will look like:</p>
<p><figure>
    <img src="../../assets/images/platform.jpg" width="600" alt="ELIXIR Belgium">
    <figcaption>JupyterLab interface.</figcaption>
</figure></p>
</details>
<h2 id="12-elixir-belgium-node-services-used-in-this-training">1.2 ELIXIR Belgium node services used in this training</h2>
<p>Belgium is part of the ELIXIR Europe network as a National Node. The Belgian node, <a href="https://www.elixir-belgium.org/">ELIXIR Belgium</a>, provides both federal-level services and local initiatives, including research infrastructure, domain-specific services, training activities, and workshops.</p>
<figure>
    <img src="../../assets/images/ELIXIR_BELGIUM.png" width="200" alt="ELIXIR Belgium">
    <figcaption>ELIXIR Belgium</figcaption>
</figure>

<p>This course focuses on two bioinformatics tools and resources: <strong>B2BTools</strong>, used to predict and analyse protein biophysical features, and <strong>Scop3P</strong>, used to explore post-translational modifications (PTMs).</p>
<h3 id="121-introduction-to-bio2byte-tools">1.2.1 Introduction to Bio2Byte Tools</h3>
<p><strong>DynaMine</strong> <sup id="fnref:dynamine"><a class="footnote-ref" href="#fn:dynamine">1</a></sup> is a predictor specifically designed to estimate protein backbone dynamics. Backbone dynamics are related to, but not the same as, protein disorder. DynaMine was trained using values derived from NMR chemical shift data and therefore captures protein movements in solution. Its training set includes both fully folded proteins and intrinsically disordered proteins.</p>
<p>The <strong>B2BTools</strong> tool suite <sup id="fnref:b2btools"><a class="footnote-ref" href="#fn:b2btools">2</a></sup> extends the original DynaMine predictor by including several predictors developed by the Bio2Byte lab.</p>
<p>In addition to backbone dynamics, it provides predictions for side-chain dynamics and conformational preferences (alpha helix, beta sheet, and coil), all derived from NMR data and trained using the same methodology. The platform also includes predictors for early folding regions (<strong>EFoldMine</strong>) <sup id="fnref:efoldmine"><a class="footnote-ref" href="#fn:efoldmine">3</a></sup>, beta-sheet aggregation (<strong>AgMata</strong>) <sup id="fnref:agmata"><a class="footnote-ref" href="#fn:agmata">4</a></sup>, and protein disorder (<strong>DisoMine</strong>) <sup id="fnref:disomine"><a class="footnote-ref" href="#fn:disomine">5</a></sup>.</p>
<h3 id="122-introduction-to-scop3p-and-scop3ptm">1.2.2 Introduction to Scop3P (and Scop3PTM)</h3>
<p>Scop3P <sup id="fnref:Scop3P"><a class="footnote-ref" href="#fn:Scop3P">6</a></sup>, developed at Ghent University and available online since June 2019, is a dedicated resource to explore and interpret the impact of phosphorylation sites on human protein structure and function. It supports researchers in analysing individual phosphosites or phosphoproteins within a structural, biophysical, and biological context.</p>
<p>The resource integrates public data from several major international databases, including UniProtKB and the Protein Data Bank (PDB). In addition, it incorporates reprocessed mass spectrometry-based phosphoproteomics data from PRIDE/ProteomeXchange. These datasets are collected worldwide, making Scop3P a strongly international and community-driven resource.</p>
<div class="admonition success">
<p class="admonition-title">About the future of Scop3P and the development of Scop3PTM</p>
<p><strong>Scop3P is being extended to Scop3PTM</strong></p>
<ul>
<li>Scop3PTM integrates information from diﬀerent knowledge bases and shows how re-analysis of large scale public proteomics data sets can add an additional level of significance and confidence to the PTM-sites.</li>
<li>Scop3PTM system will provide a unique and powerful resource to understand the impact of PTM-sites on human protein structure-function relationship.</li>
</ul>
<p>Beta access is available at <a href="https://iomics.ugent.be/scop3ptm/">https://iomics.ugent.be/scop3ptm/</a>.</p>
</div>
<hr>
<div class="admonition note">
<p class="admonition-title">Let’s get started: Go to chapter 2</p>
<p><a href="/../../chapters/chapter_02">Next chapter</a> explains the biological context.</p>
</div>
<h2 id="13-rethinking-the-protein-structure-and-function">1.3 Rethinking the protein structure and function</h2>
<p>From “Assessing the relation between protein phosphorylation, alphafold3 models, and conformational variability” <sup id="fnref:ptms-af3"><a class="footnote-ref" href="#fn:ptms-af3">7</a></sup>:</p>
<blockquote>
<p>Proteins perform diverse functions critical to cellular processes. Transitions between functional states are often regulated by post-translational modifications (PTMs) such as phosphorylation, which dynamically influence protein structure, function, folding, and interactions. Dysregulation of PTMs can therefore contribute to diseases such as cancer and Alzheimer’s. However, the structure–function relationship between proteins and their modifications remains poorly understood due to a lack of experimental structural data, the inherent diversity of PTMs, and the dynamic nature of proteins. </p>
<p>Recent advances in deep learning, particularly AlphaFold, have transformed protein structure prediction with near-experimental accuracy. However, it remains unclear whether these models can effectively capture PTM-driven conformational changes, such as those induced by phosphorylation. Here, we systematically evaluated AlphaFold models (AF2, AF3-non phospho, and AF3-phospho) to assess their ability to predict phosphorylation-induced structural diversity. By analyzing experimentally derived conformational ensembles, we found that all models predominantly aligned with dominant structural states, often failing to capture phosphorylation-specific conformations. Despite its phosphorylation-aware design, AF3-phospho predictions provided only modest improvement over AF2 and AF3-non phospho predictions. </p>
<p>Our findings highlight key challenges in modeling PTM-driven structural landscapes and underscore the need for more adaptable structure prediction frameworks capable of capturing modification-induced conformational variability.</p>
</blockquote>
<h3 id="131-proteins-are-dynamic-systems-not-static-objects">1.3.1 Proteins are dynamic systems, not static objects</h3>
<figure>
    <img src="../../assets/images/structure_memory2.png" width="800" alt="Image on elixir landing page">
</figure>

<h3 id="132-limitations-of-deep-learningbased-structure-prediction">1.3.2 Limitations of deep learning–based structure prediction</h3>
<figure>
    <img src="../../assets/images/Protein%20coverage.png" width="800" alt="Image on elixir landing page">
</figure>

<figure>
    <img src="../../assets/images/struct_mem1.png" width="800" alt="Image on elixir landing page">
</figure>

<h3 id="133-memorization-and-dominant-conformations">1.3.3 Memorization and dominant conformations</h3>
<figure>
    <img src="../../assets/images/Sturct_mem3.png" width="800" alt="Image on elixir landing page">
</figure>

<div class="footnote">
<hr>
<ol>
<li id="fn:dynamine">
<p>Elisa Cilia, Rita Pancsa, Peter Tompa, Tom Lenaerts, and Wim F Vranken. From protein sequence to dynamics and disorder with dynamine. <em>Nature communications</em>, 4:2741, 2013. URL: <a href="https://doi.org/10.1038/ncomms3741">https://doi.org/10.1038/ncomms3741</a>, <a href="https://doi.org/10.1038/ncomms3741">doi:10.1038/ncomms3741</a>. <a class="footnote-backref" href="#fnref:dynamine" title="Jump back to footnote 1 in the text">↩</a></p>
</li>
<li id="fn:b2btools">
<p>Jose Gavalda-Garcia, Adrián Díaz, and Wim Vranken. Bio2byte tools deployment as a python package and galaxy tool to predict protein biophysical properties. <em>Bioinformatics (Oxford, England)</em>, 40(9):btae543, September 2024. URL: <a href="https://europepmc.org/articles/PMC11873786">https://europepmc.org/articles/PMC11873786</a>, <a href="https://doi.org/10.1093/bioinformatics/btae543">doi:10.1093/bioinformatics/btae543</a>. <a class="footnote-backref" href="#fnref:b2btools" title="Jump back to footnote 2 in the text">↩</a></p>
</li>
<li id="fn:efoldmine">
<p>Daniele Raimondi, Gabriele Orlando, Rita Pancsa, Taushif Khan, and Wim F Vranken. Exploring the sequence-based prediction of folding initiation sites in proteins. <em>Scientific reports</em>, 7(1):8826, August 2017. URL: <a href="https://europepmc.org/articles/PMC5562875">https://europepmc.org/articles/PMC5562875</a>, <a href="https://doi.org/10.1038/s41598-017-08366-3">doi:10.1038/s41598-017-08366-3</a>. <a class="footnote-backref" href="#fnref:efoldmine" title="Jump back to footnote 3 in the text">↩</a></p>
</li>
<li id="fn:agmata">
<p>Gabriele Orlando, Alexandra Silva, Sandra Macedo-Ribeiro, Daniele Raimondi, and Wim Vranken. Accurate prediction of protein beta-aggregation with generalized statistical potentials. <em>Bioinformatics</em>, 36(7):2076–2081, 12 2019. URL: <a href="https://doi.org/10.1093/bioinformatics/btz912">https://doi.org/10.1093/bioinformatics/btz912</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/bioinformatics/article-pdf/36/7/2076/50670159/bioinformatics_36_7_2076.pdf">arXiv:https://academic.oup.com/bioinformatics/article-pdf/36/7/2076/50670159/bioinformatics_36_7_2076.pdf</a>, <a href="https://doi.org/10.1093/bioinformatics/btz912">doi:10.1093/bioinformatics/btz912</a>. <a class="footnote-backref" href="#fnref:agmata" title="Jump back to footnote 4 in the text">↩</a></p>
</li>
<li id="fn:disomine">
<p>Gabriele Orlando, Daniele Raimondi, Francesco Codicè, Francesco Tabaro, and Wim Vranken. Prediction of disordered regions in proteins with recurrent neural networks and protein dynamics. <em>Journal of Molecular Biology</em>, 434(12):167579, 2022. URL: <a href="https://www.sciencedirect.com/science/article/pii/S0022283622001590">https://www.sciencedirect.com/science/article/pii/S0022283622001590</a>, <a href="https://doi.org/https://doi.org/10.1016/j.jmb.2022.167579">doi:https://doi.org/10.1016/j.jmb.2022.167579</a>. <a class="footnote-backref" href="#fnref:disomine" title="Jump back to footnote 5 in the text">↩</a></p>
</li>
<li id="fn:Scop3P">
<p>Pathmanaban Ramasamy, Demet Turan, Natalia Tichshenko, Niels Hulstaert, Elien Vandermarliere, Wim Vranken, and Lennart Martens. Scop3p: a comprehensive resource of human phosphosites within their full context. <em>Journal of proteome research</em>, 19(8):3478—3486, August 2020. URL: <a href="https://doi.org/10.1021/acs.jproteome.0c00306">https://doi.org/10.1021/acs.jproteome.0c00306</a>, <a href="https://doi.org/10.1021/acs.jproteome.0c00306">doi:10.1021/acs.jproteome.0c00306</a>. <a class="footnote-backref" href="#fnref:Scop3P" title="Jump back to footnote 6 in the text">↩</a></p>
</li>
<li id="fn:ptms-af3">
<p>Pathmanaban Ramasamy, Jasper Zuallaert, Lennart Martens, and Wim F Vranken. Assessing the relation between protein phosphorylation, alphafold3 models, and conformational variability. <em>Protein science : a publication of the Protein Society</em>, 35(1):e70376, January 2026. URL: <a href="https://europepmc.org/articles/PMC12723721">https://europepmc.org/articles/PMC12723721</a>, <a href="https://doi.org/10.1002/pro.70376">doi:10.1002/pro.70376</a>. <a class="footnote-backref" href="#fnref:ptms-af3" title="Jump back to footnote 7 in the text">↩</a></p>
</li>
</ol>
</div></div>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
    
      
      <nav class="md-footer__inner md-grid" aria-label="Footer" >
        
          
          <a href="../../course_schedule/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Course schedule">
            <div class="md-footer__button md-icon">
              
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
            </div>
            <div class="md-footer__title">
              <span class="md-footer__direction">
                Previous
              </span>
              <div class="md-ellipsis">
                Course schedule
              </div>
            </div>
          </a>
        
        
          
          <a href="../chapter_02/" class="md-footer__link md-footer__link--next" aria-label="Next: 2. Introducing the biological context">
            <div class="md-footer__title">
              <span class="md-footer__direction">
                Next
              </span>
              <div class="md-ellipsis">
                2. Introducing the biological context
              </div>
            </div>
            <div class="md-footer__button md-icon">
              
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11z"/></svg>
            </div>
          </a>
        
      </nav>
    
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Template made by the ELIXIR Training Platform. Training content written by Pathmanaban Ramasamy and Adrián Díaz. This is a VIB Training. ELIXIR Belgium is a partner of this course.
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "../..", "features": ["navigation.top", "navigation.footer", "toc.integrate"], "search": "../../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.79ae519e.min.js"></script>
      
    
  </body>
</html>