
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../../follow_up_training/">
      
      
      
        
      
      
      <link rel="icon" href="../../assets/images/elixir_favicon.ico">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.1">
    
    
      
        <title>References - Assess protein dynamics & post-translational modifications</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.484c7ddc.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Lato";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../stylesheets/extra.css">
    
      <link rel="stylesheet" href="../../stylesheets/neoteroi-mkdocs.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
        <aside class="md-banner">
          <div class="md-banner__inner md-grid md-typeset">
            
            
This training material is under active development and considered pre-alpha.

          </div>
          
        </aside>
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Assess protein dynamics &amp; post-translational modifications" class="md-header__button md-logo" aria-label="Assess protein dynamics & post-translational modifications" data-md-component="logo">
      
  <img src="../../assets/images/white-orange-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Assess protein dynamics & post-translational modifications
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              References
            
          </span>
        </div>
      </div>
    </div>
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/Bio2Byte/training-protein-dynamics-post-translational-modifications" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    Bio2Byte/training-protein-dynamics-post-translational-modifications
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



  

<nav class="md-nav md-nav--primary md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Assess protein dynamics &amp; post-translational modifications" class="md-nav__button md-logo" aria-label="Assess protein dynamics & post-translational modifications" data-md-component="logo">
      
  <img src="../../assets/images/white-orange-logo.png" alt="logo">

    </a>
    Assess protein dynamics & post-translational modifications
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Bio2Byte/training-protein-dynamics-post-translational-modifications" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    Bio2Byte/training-protein-dynamics-post-translational-modifications
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    About
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../course_schedule/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Course schedule
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Course Content
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Course Content
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_01/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    1. Introducing this course
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_02/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    2. Introducing the biological context
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_03/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    3. Building biological profiles
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_04/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    4. Working with post-translational modifications
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_05/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    5. Combining PTM annotations to biophysical properties
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapter_06/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    6. Closing
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../follow_up_training/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Follow up training
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    
  
    References
  

    
  </span>
  
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
                  


  
  


  <h1>References</h1>

<div class="footnote">
<hr>
<ol>
<li id="fn:efoldmine">
<p>Daniele Raimondi, Gabriele Orlando, Rita Pancsa, Taushif Khan, and Wim F Vranken. Exploring the sequence-based prediction of folding initiation sites in proteins. <em>Scientific reports</em>, 7(1):8826, August 2017. URL: <a href="https://europepmc.org/articles/PMC5562875">https://europepmc.org/articles/PMC5562875</a>, <a href="https://doi.org/10.1038/s41598-017-08366-3">doi:10.1038/s41598-017-08366-3</a>. <a class="footnote-backref" href="#fnref:efoldmine" title="Jump back to footnote 1 in the text">↩</a></p>
</li>
<li id="fn:agmata">
<p>Gabriele Orlando, Alexandra Silva, Sandra Macedo-Ribeiro, Daniele Raimondi, and Wim Vranken. Accurate prediction of protein beta-aggregation with generalized statistical potentials. <em>Bioinformatics</em>, 36(7):2076–2081, 12 2019. URL: <a href="https://doi.org/10.1093/bioinformatics/btz912">https://doi.org/10.1093/bioinformatics/btz912</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/bioinformatics/article-pdf/36/7/2076/50670159/bioinformatics_36_7_2076.pdf">arXiv:https://academic.oup.com/bioinformatics/article-pdf/36/7/2076/50670159/bioinformatics_36_7_2076.pdf</a>, <a href="https://doi.org/10.1093/bioinformatics/btz912">doi:10.1093/bioinformatics/btz912</a>. <a class="footnote-backref" href="#fnref:agmata" title="Jump back to footnote 2 in the text">↩</a></p>
</li>
<li id="fn:shiftcrypt">
<p>Gabriele Orlando, Daniele Raimondi, Luciano Porto Kagami, and Wim F Vranken. Shiftcrypt: a web server to understand and biophysically align proteins through their nmr chemical shift values. <em>Nucleic Acids Research</em>, 48(W1):W36–W40, 05 2020. URL: <a href="https://doi.org/10.1093/nar/gkaa391">https://doi.org/10.1093/nar/gkaa391</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/nar/article-pdf/48/W1/W36/33432832/gkaa391.pdf">arXiv:https://academic.oup.com/nar/article-pdf/48/W1/W36/33432832/gkaa391.pdf</a>, <a href="https://doi.org/10.1093/nar/gkaa391">doi:10.1093/nar/gkaa391</a>. <a class="footnote-backref" href="#fnref:shiftcrypt" title="Jump back to footnote 3 in the text">↩</a></p>
</li>
<li id="fn:psper">
<p>Gabriele Orlando, Daniele Raimondi, Francesco Tabaro, Francesco Codicè, Yves Moreau, and Wim F Vranken. Computational identification of prion-like rna-binding proteins that form liquid phase-separated condensates. <em>Bioinformatics</em>, 35(22):4617–4623, 04 2019. URL: <a href="https://doi.org/10.1093/bioinformatics/btz274">https://doi.org/10.1093/bioinformatics/btz274</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/bioinformatics/article-pdf/35/22/4617/48978636/bioinformatics_35_22_4617.pdf">arXiv:https://academic.oup.com/bioinformatics/article-pdf/35/22/4617/48978636/bioinformatics_35_22_4617.pdf</a>, <a href="https://doi.org/10.1093/bioinformatics/btz274">doi:10.1093/bioinformatics/btz274</a>. <a class="footnote-backref" href="#fnref:psper" title="Jump back to footnote 4 in the text">↩</a></p>
</li>
<li id="fn:disomine">
<p>Gabriele Orlando, Daniele Raimondi, Francesco Codicè, Francesco Tabaro, and Wim Vranken. Prediction of disordered regions in proteins with recurrent neural networks and protein dynamics. <em>Journal of Molecular Biology</em>, 434(12):167579, 2022. URL: <a href="https://www.sciencedirect.com/science/article/pii/S0022283622001590">https://www.sciencedirect.com/science/article/pii/S0022283622001590</a>, <a href="https://doi.org/https://doi.org/10.1016/j.jmb.2022.167579">doi:https://doi.org/10.1016/j.jmb.2022.167579</a>. <a class="footnote-backref" href="#fnref:disomine" title="Jump back to footnote 5 in the text">↩</a></p>
</li>
<li id="fn:b2btools">
<p>Jose Gavalda-Garcia, Adrián Díaz, and Wim Vranken. Bio2byte tools deployment as a python package and galaxy tool to predict protein biophysical properties. <em>Bioinformatics (Oxford, England)</em>, 40(9):btae543, September 2024. URL: <a href="https://europepmc.org/articles/PMC11873786">https://europepmc.org/articles/PMC11873786</a>, <a href="https://doi.org/10.1093/bioinformatics/btae543">doi:10.1093/bioinformatics/btae543</a>. <a class="footnote-backref" href="#fnref:b2btools" title="Jump back to footnote 6 in the text">↩</a></p>
</li>
<li id="fn:dynamine">
<p>Elisa Cilia, Rita Pancsa, Peter Tompa, Tom Lenaerts, and Wim F Vranken. From protein sequence to dynamics and disorder with dynamine. <em>Nature communications</em>, 4:2741, 2013. URL: <a href="https://doi.org/10.1038/ncomms3741">https://doi.org/10.1038/ncomms3741</a>, <a href="https://doi.org/10.1038/ncomms3741">doi:10.1038/ncomms3741</a>. <a class="footnote-backref" href="#fnref:dynamine" title="Jump back to footnote 7 in the text">↩</a></p>
</li>
<li id="fn:b2btools-webserver">
<p>Luciano Porto Kagami, Gabriele Orlando, Daniele Raimondi, Francois Ancien, Bhawna Dixit, Jose Gavaldá-García, Pathmanaban Ramasamy, Joel Roca-Martínez, Konstantina Tzavella, and Wim Vranken. B2btools: online predictions for protein biophysical features and their conservation. <em>Nucleic Acids Research</em>, 49(W1):W52–W59, 05 2021. URL: <a href="https://doi.org/10.1093/nar/gkab425">https://doi.org/10.1093/nar/gkab425</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/nar/article-pdf/49/W1/W52/38842326/gkab425.pdf">arXiv:https://academic.oup.com/nar/article-pdf/49/W1/W52/38842326/gkab425.pdf</a>, <a href="https://doi.org/10.1093/nar/gkab425">doi:10.1093/nar/gkab425</a>. <a class="footnote-backref" href="#fnref:b2btools-webserver" title="Jump back to footnote 8 in the text">↩</a></p>
</li>
<li id="fn:Scop3P">
<p>Pathmanaban Ramasamy, Demet Turan, Natalia Tichshenko, Niels Hulstaert, Elien Vandermarliere, Wim Vranken, and Lennart Martens. Scop3p: a comprehensive resource of human phosphosites within their full context. <em>Journal of proteome research</em>, 19(8):3478—3486, August 2020. URL: <a href="https://doi.org/10.1021/acs.jproteome.0c00306">https://doi.org/10.1021/acs.jproteome.0c00306</a>, <a href="https://doi.org/10.1021/acs.jproteome.0c00306">doi:10.1021/acs.jproteome.0c00306</a>. <a class="footnote-backref" href="#fnref:Scop3P" title="Jump back to footnote 9 in the text">↩</a></p>
</li>
<li id="fn:ptms-af3">
<p>Pathmanaban Ramasamy, Jasper Zuallaert, Lennart Martens, and Wim F Vranken. Assessing the relation between protein phosphorylation, alphafold3 models, and conformational variability. <em>Protein science : a publication of the Protein Society</em>, 35(1):e70376, January 2026. URL: <a href="https://europepmc.org/articles/PMC12723721">https://europepmc.org/articles/PMC12723721</a>, <a href="https://doi.org/10.1002/pro.70376">doi:10.1002/pro.70376</a>. <a class="footnote-backref" href="#fnref:ptms-af3" title="Jump back to footnote 10 in the text">↩</a></p>
</li>
<li id="fn:MolStar">
<p>David Sehnal, Sebastian Bittrich, Mandar Deshpande, Radka Svobodová, Karel Berka, Václav Bazgier, Sameer Velankar, Stephen K Burley, Jaroslav Koča, and Alexander S Rose. Mol* viewer: modern web app for 3d visualization and analysis of large biomolecular structures. <em>Nucleic Acids Research</em>, 49(W1):W431–W437, 05 2021. URL: <a href="https://doi.org/10.1093/nar/gkab314">https://doi.org/10.1093/nar/gkab314</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/nar/article-pdf/49/W1/W431/38842088/gkab314.pdf">arXiv:https://academic.oup.com/nar/article-pdf/49/W1/W431/38842088/gkab314.pdf</a>, <a href="https://doi.org/10.1093/nar/gkab314">doi:10.1093/nar/gkab314</a>. <a class="footnote-backref" href="#fnref:MolStar" title="Jump back to footnote 11 in the text">↩</a></p>
</li>
<li id="fn:MolViewSpec">
<p>Adam Midlik, Sebastian Bittrich, Jennifer R Fleming, Sreenath Nair, Sameer Velankar, Stephen K Burley, Jasmine Y Young, Brinda Vallat, and David Sehnal. Molviewspec: a mol* extension for describing and sharing molecular visualizations. <em>Nucleic Acids Research</em>, 53(W1):W408–W414, 05 2025. URL: <a href="https://doi.org/10.1093/nar/gkaf370">https://doi.org/10.1093/nar/gkaf370</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/nar/article-pdf/53/W1/W408/63067309/gkaf370.pdf">arXiv:https://academic.oup.com/nar/article-pdf/53/W1/W408/63067309/gkaf370.pdf</a>, <a href="https://doi.org/10.1093/nar/gkaf370">doi:10.1093/nar/gkaf370</a>. <a class="footnote-backref" href="#fnref:MolViewSpec" title="Jump back to footnote 12 in the text">↩</a></p>
</li>
<li id="fn:UniProtConsortium">
<p>The UniProt Consortium. Uniprot: the universal protein knowledgebase in 2025. <em>Nucleic Acids Research</em>, 53(D1):D609–D617, 11 2024. URL: <a href="https://doi.org/10.1093/nar/gkae1010">https://doi.org/10.1093/nar/gkae1010</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/nar/article-pdf/53/D1/D609/60719276/gkae1010.pdf">arXiv:https://academic.oup.com/nar/article-pdf/53/D1/D609/60719276/gkae1010.pdf</a>, <a href="https://doi.org/10.1093/nar/gkae1010">doi:10.1093/nar/gkae1010</a>. <a class="footnote-backref" href="#fnref:UniProtConsortium" title="Jump back to footnote 13 in the text">↩</a></p>
</li>
<li id="fn:geert_van_geest_2023_7913092">
<p>Geert van Geest, Elin Kronander, Jose Alejandro Romero Herrera, Nadja Žlender, ELIXIR Training Coordination Team, and Alexia Cardona. The elixir training lesson template - developing training together. May 2023. URL: <a href="https://doi.org/10.5281/zenodo.7913092">https://doi.org/10.5281/zenodo.7913092</a>, <a href="https://doi.org/10.5281/zenodo.7913092">doi:10.5281/zenodo.7913092</a>. <a class="footnote-backref" href="#fnref:geert_van_geest_2023_7913092" title="Jump back to footnote 14 in the text">↩</a></p>
</li>
</ol>
</div>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
    
      
      <nav class="md-footer__inner md-grid" aria-label="Footer" >
        
          
          <a href="../../follow_up_training/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Follow up training">
            <div class="md-footer__button md-icon">
              
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
            </div>
            <div class="md-footer__title">
              <span class="md-footer__direction">
                Previous
              </span>
              <div class="md-ellipsis">
                Follow up training
              </div>
            </div>
          </a>
        
        
      </nav>
    
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Template made by the ELIXIR Training Platform. Training content written by Pathmanaban Ramasamy and Adrián Díaz. This is a VIB Training. ELIXIR Belgium is a partner of this course.
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "../..", "features": ["navigation.top", "navigation.footer", "toc.integrate"], "search": "../../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.79ae519e.min.js"></script>
      
    
  </body>
</html>